src/book/book.rs

```diff
@@ -14,11 +14,12 @@ pub fn load_book<P: AsRef<Path>>(src_dir: P, cfg: &BuildConfig) -> Result<Book>
     let summary_md = src_dir.join("SUMMARY.md");
 
     let mut summary_content = String::new();
-    File::open(summary_md)
-        .with_context(|| "Couldn't open SUMMARY.md")?
+    File::open(summary_md.clone())
```

Comment:
The clone here isn't necessary, it can just take a reference.

```suggestion
    File::open(&summary_md)
```

Reply:
Thank you for the feedback. Will do the necessary changes.